app/services/make_an_offer.rb

```diff
@@ -20,4 +24,18 @@ def save
     errors.add(:state, e.message)
     false
   end
+
+private
+
+  def offer_conditions_valid?
```

Comment:
The name of this methods implies that it check whether `makeanoffer` is valid, but it isn't the case.
`valid?`(line 12) is the one that does it, and it just checks if `errors` is empty? 

Should this be renamed to `validate_condition`? (or similar)

Reply:
I don't think I follow — wouldn't `make_an_offer_valid?` imply whether this `MakeAnOffer` is valid? If this is confusing, though, it should be changed

Reply:
`validate_offer_conditions` sounds good!